README.md

```diff
@@ -0,0 +1,113 @@
+    * Bird name [`name`]
+	* Migratory cycle [`cycle`] (winter, spring, summer, autumn + year)
+	* Starting date [`date`]
+    * Behaviour type [`type`] (breeding, dead, migration, move, pre-migration, winter.stage1, winter.stage2, winter.stage3)
+    * Criteria for selection in analysis ([`complete`], [`use.date`])
+* annotation-swedes.csv: annotated version of the Swedish dataset. Wouter Vansteelant determined the start and end dates of migratory and non-migratory periods through expert interpretation of tracks in Google Earth.
+    * Bird name [`name`]
+	* Migratory cycle [`cycle`] (winter, spring, summer, autumn + year)
+	* Starting date [`date`]
+    * Behaviour type [`type`] (breeding, dead, migration, move, pre-migration, winter.stage1, winter.stage2)
+    * Criteria for selection in analysis ([`complete`], [`use.date`])
+* MH-resampled-23092017.csv: Belgian and Dutch tracking data downloaded from the UvA-BiTS database on September 23rd, 2017
+	* Device and individual code ([`dev`], [`name`]
+	* Coordinates ([`lat`], [`lon`], [`alt`])
+	* Date and time
+		* Date time [`dt`]
+		* Year [`yr`]
+		* [`season`] (spring/autumn)
+		* Julian time [`julian`]
+		* day of year [`yday`]
+	* Data provider [`origin`]
+	* Derived codes ([`indday`] (name_julian), [`migr`] (name_yr_season))
+	* Calculated parameters using functions in script [`pt2pt_fxns`]
+		* Distance [`dist`]
+		* Duration [`dur`]
+		* Speed ([`spd`] = [`dist`]/[`dur`])		 
+* swedes.csv: Swedish tracking data
+* weather.csv: daily tailwind conditions calculated using the method of [Klaassen et al. (2010)](https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1600-048X.2010.05058.x). u- and v-wind components were obtained from the NOAA NCEP global atmospheric reanalysis model.
+
+#### External GIS data (available through web links)
+* ne_50m_admin_0_countries: shapefile containing country boundaries, source: Natural Earth ( https://www.naturalearthdata.com/downloads/50m-cultural-vectors/50m-admin-0-countries-2/)
+* alt_30s_bil: raster file containing a DEM of the world, source: worldclim, version 1.4 (http://biogeo.ucdavis.edu/data/climate/worldclim/1_4/grid/cur/alt_30s_bil.zip)
+
+## Scripts
+### Main scripts
+The scripts are supposed to be run consecutively.
+* p2a_analyses_read_processed_data.R: read data files
+* p2b_analyses_annotate&clean_processed_data.R: clean data
+* p3_Fig1A_Lat-vs-yday.R: make figure 1A of the paper
+* p3a_Fig1B&C.R: make figures 1B and 1C
+* p3b_FigS2_Wintering-fin.R: make figure S2
+* p4_wind-fin.R: read wind data and calculate mean daily wind components
+* p5_CompileData-PerTrip-PerID_TableS1&S9.R: make table S1 and S9
+* p6_Figure2&TablesS5-S6.R: make table S5 and S6
+* p6_TableS2.R: make table S2
+* p6a_TableS3_Long-diff-pops.R: make table S3
+* p6b_TablesS7&8_WindSelectivity_glmms&posthoc.R: make table S7 and S8
+* pt2pt_fxns.R: functions to compute distance, duration and speed
```

Comment:
On the naming of the files:

- Will the page number reference still work when the paper is formatted as journal paper?
- could you make the naming more consistent (all lowercase, `Fig1A` versus `Figure2`